src/models/EmissionFactor.ts

```diff
@@ -30,6 +30,10 @@ export type EmissionFactor = {
      */
     name: string
     source: EmissionFactorSource
+    /**
+     * Source version of the emission factor.
+     */
+    sourceVersion?: string | null
```

Comment:
@raguiar9080 something seems wrong to me, both `undefined` and `null` are allowed.

Reply:
Ah, you're correct, it doesn't need the `null` option at all. Fixing.

Reply:
https://github.com/lune-climate/lune-backend/pull/4819